apps/emqx_mt/src/emqx_mt_state.erl

```diff
@@ -56,10 +58,12 @@
     bulk_update_root_configs_txn/1
 ]).
 
-%% Internal exports for `emqx_mt_config`.
+%% Internal exports for `emqx_mt_config{,_janitor}`.
 -export([
     tables_to_backup/0,
-    ensure_ns_added/1
+    ensure_ns_added/1,
+    clear_tombstone/1,
+    is_tombstoned/1
```

Comment:
this API should be called before creating a namespace ?

Reply:
it's checked directly in the transaction to create the namespace.

Reply:
> this API should be called before creating a namespace ?

I mean, should not allow to add back a namespace which is being deleted.